### PR TITLE
Content updates

### PR DESCRIPTION
**Pull request by richmccartney**
#### What's this PR do?
- Amends footer content
- Updates animated GIF on ```index.html```
- Updates the content in ```try-landscape.html```
- Adds a new strapline to the try landscape row on ```index.html```
- Fixes missing links
- Adds read me content

#### Where should the reviewer start?
Pull down the branch and run ```npm install``` & ```gulp build```

#### How should this be manually tested?
View all the pages and reference the content changes against the google docs 

- https://docs.google.com/document/d/1EkwNoH23aSHOjKW0j96n7dg0yKD9uNL8Q8On7sY9iMQ
- https://docs.google.com/document/d/1UXiQtNEKOM1TjrA-oiBj_wwkOeaJJTRGi3zyjZhKPtc

Check all Scss is linting correctly by running ```gulp test```

#### Any background context you want to provide?
_n/a_

#### What are the relevant tickets?
#104 

#### Screenshots (if appropriate)
_n/a_

#### Questions:
_n/a_

### CODE REVIEW

File: contact-us.html [hunk 1]
```diff
@@ -411,19 +411,19 @@ <h3 class="footer__title">Landscape</h3>
                 <div class="five-col no-margin--bottom">
                     <h3 class="footer__title">Need help?</h3>
                     <div class="footer__second-nav">
-                        <p>Ubuntu Advantage is the professional support package from the experts at Canonical, helping organisations around the world to manage their Ubuntu deployments.</p>
-                        <p><a href="/contact-us.html" class="button--primary">Contact us</a></p>
-                        <p><a href="#" class="external">Learn more about Ubuntu advantage</a></p>
+                        <p>Landscape is part of Ubuntu Advantage, the professional support package form Canonical for managing Ubuntu deployments</p>
```
Comment (anthonydillon) [inline contact-us.html]: I think you mean from not form.

File: features.html [hunk 2]
```diff
@@ -225,19 +225,19 @@ <h3 class="footer__title">Landscape</h3>
                 <div class="five-col no-margin--bottom">
                     <h3 class="footer__title">Need help?</h3>
                     <div class="footer__second-nav">
-                        <p>Ubuntu Advantage is the professional support package from the experts at Canonical, helping organisations around the world to manage their Ubuntu deployments.</p>
-                        <p><a href="/contact-us.html" class="button--primary">Contact us</a></p>
-                        <p><a href="#" class="external">Learn more about Ubuntu advantage</a></p>
+                        <p>Landscape is part of Ubuntu Advantage, the professional support package form Canonical for managing Ubuntu deployments</p>
```
Comment (anthonydillon) [inline features.html]: from instead of form?

File: index.html [hunk 3]
```diff
@@ -262,19 +262,19 @@ <h3 class="footer__title">Landscape</h3>
                 <div class="five-col no-margin--bottom">
                     <h3 class="footer__title">Need help?</h3>
                     <div class="footer__second-nav">
-                        <p>Ubuntu Advantage is the professional support package from the experts at Canonical, helping organisations around the world to manage their Ubuntu deployments.</p>
-                        <p><a href="/contact-us.html" class="button--primary">Contact us</a></p>
-                        <p><a href="#" class="external">Learn more about Ubuntu advantage</a></p>
+                        <p>Landscape is part of Ubuntu Advantage, the professional support package form Canonical for managing Ubuntu deployments</p>
```
Comment (anthonydillon) [inline index.html]: From instead of form?

File: static/scss/base/_typography.scss [hunk 4]
```diff
@@ -32,6 +32,10 @@
     font-size: 32px;
   }
 
+  .font-medium-lrg {
+    font-size: 23px;
```
Comment (anthonydillon) [inline static/scss/base/_typography.scss]: We should use `em`'s for font sizes and only have a px value on the body.

File: thank-you.html [hunk 5]
```diff
@@ -115,19 +115,19 @@ <h3 class="footer__title">Landscape</h3>
                 <div class="five-col no-margin--bottom">
                     <h3 class="footer__title">Need help?</h3>
                     <div class="footer__second-nav">
-                        <p>Ubuntu Advantage is the professional support package from the experts at Canonical, helping organisations around the world to manage their Ubuntu deployments.</p>
-                        <p><a href="/contact-us.html" class="button--primary">Contact us</a></p>
-                        <p><a href="#" class="external">Learn more about Ubuntu advantage</a></p>
+                        <p>Landscape is part of Ubuntu Advantage, the professional support package form Canonical for managing Ubuntu deployments</p>
```
Comment (anthonydillon) [inline thank-you.html]: From instead of form?

File: try-landscape.html [hunk 6]
```diff
@@ -269,19 +270,19 @@ <h3 class="footer__title">Landscape</h3>
                 <div class="five-col no-margin--bottom">
                     <h3 class="footer__title">Need help?</h3>
                     <div class="footer__second-nav">
-                        <p>Ubuntu Advantage is the professional support package from the experts at Canonical, helping organisations around the world to manage their Ubuntu deployments.</p>
-                        <p><a href="/contact-us.html" class="button--primary">Contact us</a></p>
-                        <p><a href="#" class="external">Learn more about Ubuntu advantage</a></p>
+                        <p>Landscape is part of Ubuntu Advantage, the professional support package form Canonical for managing Ubuntu deployments</p>
```
Comment (anthonydillon) [inline try-landscape.html]: From instead of form?

**Comment by anthonydillon** [comment]
There is a small typo in the footer and please use `em`'s for adjusting font sizes from the base applied to body.

**Comment by richmccartney** [comment]
@anthonydillon give it another butch.

**Comment by anthonydillon** [comment]
Nothing updated?

**Comment by richmccartney** [comment]
@anthonydillon did you have a chance to scan through this again?

**Comment by anthonydillon** [comment]
Looks good :+1:

**Pull request merged**